text/0000-distribute-to-ecr.md

```diff
@@ -0,0 +1,94 @@
+# Image Distribution via ECR
+
+## Summary
+
+Amazon Elastic Container Registry (ECR) is Amazon's implementation of the [OCI Distribution Specification][distribution-spec]. The RFC proposes publishing all Paketo Buildpack, Stack, and Builder images to [ECR Public][ecr-public] in addition to DockerHub (our primary distribution channel to date).
+
+## Motivation
+
+Publishing images to ECR will improve the experience of using Paketo buildpacks in AWS, whether that is with [AWS App Runner][app-runner] as [proposed][app-runner-roadmap] in the App Runner roadmap or with platforms such as [kpack][kpack] on [tekton][tekton-task] running on [Amazon Elastic Kubernetes Service][eks] (EKS).
+
+The first way this will improve the Paketo+AWS experience is by enhancing performance. Pulling and pushing images is generally fastest when working within (e.g. ECR -> EKS) rather than across (e.g GCR -> EKS) cloud provider ecosystems. Keeping copies of the run image and builpack images in ECR should enable peak performance for build/rebase/create-builder with [cross-repo blob mounting][cross-repo-blob-mount] when the created images are also public. This may not apply to private images in ECR due to differing registry hostnames (although the ggcr team has an [experimental cross-origin blob mount proposal][cross-origin-blob-mount] that would allow registries like ECR to enable this).
+
+Second, working within a single cloud provider ecosystem allows users to take advantage of idiomatic access control mechanisms such as [node-identity][node-identity] or [workload-identity][workload-identity] based registry authentication. While this is probably unnecessary given Paketo images are public and our Dockerhub account should be exempt from rate limiting, hosting images in ECR removes any ambiguity, allowing users to work entirely within a comfortable toolchain.
+
+## Detailed Explanation
+
+The project has already been given an AWS organization funded by the CFF and we have requested the [custom alias][registry-alias] `paketo-buildpacks` for our public registry. We should take the additional step of contacting support and verifying this account so that our repositories will have a verified badge on the ECR Public Gallery (click on any [heroku buildpack][heroku-gallery] repository in the gallery for an example).
+
+A copy of all image artifacts including releases of buildpack, builder, and stack images should be published there in addition to Dockerhub.
+
+Examples:
+- `public.ecr.aws/paketo-buildpacks/builder:base`
+- `public.ecr.aws/paketo-buildpacks/builder:0.3.12-base`
```

Comment:
I think we should use only the new builder and stack image naming scheme:
- `builder-jammy-base:latest`
to avoid having to maintain the ambigiously named `builder` repo